multivac/gather_job_data.py

```diff
@@ -66,6 +66,15 @@ def detect_error(logs: str, failure_specs: list) -> (str, str):
     return 'unknown', None
 
 
+def github_time_to_unix(time: str) -> float:
```

Comment:
is `float` the right type for big integers?

Reply:
Indeed, it is https://docs.python.org/3/library/datetime.html#datetime.datetime.timestamp